db/fixtures/event_blacklists.yml

```diff
@@ -0,0 +1,46 @@
+#
+# The filtered_events section defines which events should not be added to the
+#   database.  Keys that have nil values will be filtered at the event catcher
+#   level.  Keys that do have values, in the format of a hash of object type to
+#   an array of ids, will be filtered such that they are not added to the event
+#   table.
+#
+
+# Vmware filtered events
+ManageIQ::Providers::Vmware::InfraManager:
+  - AlarmActionTriggeredEvent
+  - AlarmCreatedEvent
+  - AlarmEmailCompletedEvent
+  - AlarmEmailFailedEvent
+  - AlarmReconfiguredEvent
+  - AlarmRemovedEvent
+  - AlarmScriptCompleteEvent
+  - AlarmScriptFailedEvent
+  - AlarmSnmpCompletedEvent
+  - AlarmSnmpFailedEvent
+  - AlarmStatusChangedEvent
+  - AlreadyAuthenticatedSessionEvent
+  - EventEx
+  - UserLoginSessionEvent
+  - UserLogoutSessionEvent
+
+# RHEV filtered events
+ManageIQ::Providers::Redhat::InfraManager:
+  - UNASSIGNED
+  - USER_REMOVE_VG
+  - USER_REMOVE_VG_FAILED
+  - USER_VDC_LOGIN
+  - USER_VDC_LOGOUT
+  - USER_VDC_LOGIN_FAILED
+
+# OpenStack filtered events
+EmsOpenstack:
```

Comment:
I promise I'll forget this by the time I have to rebase.. but hopefully a test somewhere will fail.

I'm more concerned by the general fact we're introducing such an explicitly per-manager dataset, in a central location. I gather this is ultimately just step 1 of a plan... but can we push these lists into an `EmsOpenstack::DEFAULT_FILTERED_EVENTS` constant or something instead?